sqlx-core/src/sqlite/options/mod.rs

```diff
@@ -183,6 +185,14 @@ impl SqliteConnectOptions {
         self
     }
 
+    /// Sets custom initial settings for the database connection.
+    ///
+    /// The default initial_options is NONE.
+    pub fn initial_options(mut self, initial_options: &'static str) -> Self {
```

Comment:
The documentation and name for this method is really vague. It sounds like a list of `PRAGMAS` to specify but really it's a whole statement to execute.

For clarity's sake, why not give it the name `pragmas` and have it take a list of key-value pairs? Collect them internally to a vector or order-preserving map and deduplicate the keys with those we set by default.

Reply:
A `&'static str` isn't that useful here anyway, especially if you're setting a SQLCipher encryption key (which would normally be derived at runtime from a user-entered password).

Reply:
Yes you are correct, i will change it.